src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -54,6 +54,8 @@ public void update(StandaloneConfig standaloneConfig, List<String> ignoreGlobLis
             List<String> aliases = new ArrayList<>(author.getAuthorAliases());
             List<String> emails = new ArrayList<>(author.getEmails());
             aliases.add(author.getGitId());
+
+            checkDuplicateAliases(aliases);
```

Comment:
There may be an issue here (let me know if I'm wrong)

In the method `checkDuplicateAliases`, we are checking a list of aliases against the aliases present in the *current* `authorEmailsAndAliasesMap` TreeMap of the AuthorConfiguration instance.

However, in the `update` method, we are adding the aliases to a *new* TreeMap.
i.e. this will not detect duplicate aliases that occur within the standalone configuration file?

Reply:
I tried analysing a [test-repo with duplicate aliases in its standalone config](https://github.com/anubh-v/test-repo) using this PR. 
There do not seem to be any logged warnings when I run RepoSense:

Let me know if I made an error in testing.

Here is the [link](https://github.com/anubh-v/test-repo/blob/master/_reposense/config.json) to the standalone config in the repo.
There are 2 authors, `Anubhav` and `Bob` in the standalone config, both with the same alias.

Here is the output from running RepoSense:
![image](https://user-images.githubusercontent.com/35621759/77233424-42758380-6be2-11ea-8928-7b8da0992515.png)

Command used:
`./gradlew run -Dargs="--repo https://github.com/anubh-v/test-repo.git"`

Reply:
Yup, realized this too. Currently, fixing it in local. I will commit soon. 

I have found a quick fix for this. Trying to see if some better way possible.